modelci/persistence/README.md

```diff
@@ -42,7 +42,7 @@ See test `test/test_model_service.test_register_model`.
 from modelci.persistence.service import ModelService
 
 model_service = ModelService()
-models = model_service.get_models_by_name('ResNet50')
+models = model_service.get_models_by_pk('ResNet50')
```

Comment:
what  is pk？

Reply:
primary keys (commonly used in python backend as Django, etc.) We are able to get by one or more of {name, framework, engine, version (to be updated)}.

Reply:
to be honest, understanding the function from the name easily is what we pursue. 

I prefer the name, id, etc.

Reply:
in that case, the function name is too long for me `get_models_by_name_framework_engine_version`

Reply:
get_models?

Reply:
done